_posts/en/newsletters/2023-12-13-newsletter.md

```diff
@@ -0,0 +1,202 @@
+    the risk that providers may close channels shortly after receiving
+    their liquidity fees.  If channels opened through liquidity ads
+    typically generate significant forwarding fee income, there would be
+    an incentive to keep channels open.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Stratum v2 mining pool launches:**
+  [DEMAND][demand website] is a mining pool built off of the [Stratum v2
+  reference implementation][news247 sri] initially allowing for solo mining, with
+  pooled mining planned for the future.
+
+- **Bitcoin network simulation tool warnet announced:**
+  The [warnet software][warnet github] allows for specifying node topologies,
+  running [scripted scenarios][warnet scenarios] across that network, and
+  [monitoring][warnet monitoring] and analyzing resulting behaviors.
+
+- **Payjoin client for Bitcoin Core released:**
+  The [payjoin-cli][] is a rust project that adds command line [payjoin][topic payjoin] sending
+  and receiving capabilities for Bitcoin Core.
+
+- **Call for community block arrival timestamps:**
+  A contributor to the [Bitcoin Block Arrival Time Dataset][block arrival github]
+  repository [called][b10c tweet] for node operators to submit their block arrival
+  timestamps for research. There is a similar repository for collecting [stale
+  block data][stale block github].
+
+- **Envoy 1.4 released:**
+  Bitcoin wallet Envoy’s [1.4 release][envoy v1.4.0] adds [coin control][topic
+  coin selection] and [wallet labeling][topic wallet labels] ([BIP329][]
+  coming soon), among other features.
+
+- **BBQr encoding scheme announced:**
+  The [scheme][bbqr github] can efficiently encode larger files, for example [PSBTs][topic
+  psbt], into an animated QR series for use in air-gapped wallet configurations.
+
+- **Zeus v0.8.0 released:**
+  The [v0.8.0][zeus v0.8.0] release contains an embedded LND node, additional
+  [zero conf channel][topic zero-conf channels] support, and support for simple taproot channels,
+  among other changes.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aa
+nswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What are all the rules related to CPFP fee bumping?]({{bse}}120853)
+  Pieter Wuille points out that contrary to the [RBF][topic rbf] fee bumping
+  technique that has a list of associated policy rules, the [CPFP][topic cpfp]
+  fee bumping technique has no additional policy rules.
```

Comment:
This isn't a comment on the PR here, but mention of "policy" reminds me that recently I saw a useful distinction that I wasn't aware of, it may have been in a Murch tweet (sorry, post on X). Transaction "standardness" relates to properties of a single transaction considered in isolation; "policy" involves multiple transactions (ancestor and descendant counts, for example). Previously, I had thought of these two words as somewhat interchangeable. Just thought I'd pass that along in case others here weren't aware of those definitions!

Reply:
I don't think that's quite right.  I agree that standardness relates to the properties of a single transaction (inclusive of its prevouts),  but I consider policy to be a superset of standardness rules that also includes all other rules related to whether a node will accept a transaction into its mempool and relay it.  I think this is evidenced by `IsStandard()` and `IsStandardTx` being defined in `policy.*`.